src/users/tasks.py

```diff
@@ -0,0 +1,13 @@
+from django.apps import apps
+
+from app.celery import celery
+from app.tasks import update_dashamail_subscription
+from users.tags.pipeline import apply_tags
+
+
+@celery.task()
```

Comment:
```suggestion
@celery.task(name='users.rebuild_tags')
```

Так по логам будет гораздо легче отлавливать ошибки. А то сразу и не поймёшь, какие именно теги мы тут ребилдим